package.json

```diff
@@ -104,7 +105,6 @@
     "parse-svg-path": "^0.1.2",
     "polybooljs": "^1.2.0",
     "probe-image-size": "^7.2.2",
-    "regl": "^2.1.0",
```

Comment:
Why this line is removed?
It should point to 
```js
"regl": "git://github.com/yujin-wu/regl.git#29c9b09ca5f078b674131e20f3a99197550521b5",
```

Reply:
Please use 
```sh
npm install --save git://github.com/yujin-wu/regl.git#29c9b09ca5f078b674131e20f3a99197550521b5
``` 
as suggested here: https://github.com/plotly/plotly.js/pull/6083#discussion_r800803290

Reply:
Also for now let's switch back to use `node v12 and npm v6` for development here.
For strange reasons `node 16 and npm v7` require this unused [gl](https://www.npmjs.com/package/gl) module! Perhaps related to https://github.com/stackgl/headless-gl/issues/221.

Reply:
Please note that you also need to fork `gl-text` and then link to your regl patch in `gl-text` module.

Reply:
It appears at least forking gl-text is not necessary, it seems its regl dependency resolves to the custom one.

Reply:
Also for containers that rely on `node12` dist, please switch from `cimg/node:16.8.0` to `cimg/node:12.22.7`.